### PR TITLE
surface step compression

### PR DESCRIPTION
**Pull request by brownd1978**
Patch on top of PR #330 to address CI issues

### CODE REVIEW

File: JobConfig/pileup/prolog.fcl [hunk 1]
```diff
@@ -22,6 +22,7 @@ Pileup: {
       module_type : CompressDetStepMCs
       strawGasStepTag : "StrawGasStepMaker"
       caloShowerStepTag : "CaloShowerStepMaker"
+      surfaceStepTag : ""
```
Comment (AndrewEdmonds11) [inline JobConfig/pileup/prolog.fcl:25]: Thanks, Dave. I think this should be ```surfaceStepTags : [ ]```

**Review by AndrewEdmonds11 — COMMENTED** [review]
Thanks, Dave. I think just one correction noted below. Also, should I close #330 and then run Mu2e/Offline#1306 tests with this one?

File: JobConfig/pileup/prolog.fcl (hunk 1 above)
Comment (brownd1978) [reply JobConfig/pileup/prolog.fcl:25]: Hmm, it worked fine when I tested it. The code seems to require an atom, not a sequence. Maybe you're thinking of CompressDigiMCS?

**Comment by brownd1978** [comment]
@FNALbuild run build tests with Mu2e/Offline#1306

**Review by AndrewEdmonds11 — APPROVED** [review]

**Pull request merged**